srtcore/utilities.h

```diff
@@ -630,7 +636,15 @@ class UniquePtr: public std::auto_ptr<T>
     operator bool () { return 0!= get(); }
 };
 
-// A primitive one-argument version of Printable
+// A primitive one-argument versions of Sprint and Printable
+template <class Arg1>
+inline std::string Sprint(const Arg1& arg)
```

Comment:
SPrint might be better, to avoid confusion with "sprint"

Reply:
It already exists. This is a C++03 version of an existing function. That's still not the best name of this function.